stable/chronograf/values.yaml

```diff
@@ -0,0 +1,38 @@
+image:
+  repository: "quay.io/influxdb/chronograf"
+  tag: "latest"
+  pullPolicy: "Always"
+
+## Specify a service type
+## NodePort is default
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: LoadBalancer
```

Comment:
Lets make it a ClusterIP, lets err on the side of making things more private rather than more public. The port-forwarding instructions in the NOTES will work all the same.